index.js

```diff
@@ -389,10 +391,17 @@ function session(options) {
         })
       }
 
-      function save() {
+      function save(callback) {
         debug('saving %s', this.id);
         savedHash = hash(this);
-        _save.apply(this, arguments);
+        _save.call(this, function (err) {
+          if (err) {
+           return callback(err)
```

Comment:
```suggestion
            return callback(err)
```

Sorry, doing it through the GH UI is hard to count spaces 😆

Reply:
thanks, fixed it.